<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettingsWindow</class>
 <widget class="QWidget" name="AppSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSteamClient settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="libraryFoldersTab">
         <attribute name="title">
          <string>Library Folders</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
            <item>
             <widget class="QListWidget" name="libraryFolders"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="addFolderButton">
                <property name="text">
                 <string>Add Folder</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="moveContentButton">
                <property name="text">
                 <string>Move Content</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteFolderButton">
                <property name="text">
                 <string>Delete folder</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Removing folders doesn't work 100% of the time. Try disabling downloads if folder management actions fail.</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="compatibilityTab">
         <attribute name="title">
          <string>Compatibility</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3c">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="enableCompatToolsCheck">
                <property name="text">
                 <string>Enable Compatibility tools</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>If this isn't checked, you will not be able to play Windows-only games.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Needs client restart to apply.</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_4"/>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout_5"/>
          </item>
          <item row="1" column="2">
           <layout class="QGridLayout" name="gridLayout_6"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="downloadsTab">
         <attribute name="title">
          <string>Downloads</string>
         </attribute>
         <widget class="QCheckBox" name="allowDownloadsWhilePlayingCheck">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>231</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Allow downloads while playing</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
